app.R

```diff
@@ -298,6 +314,40 @@ server <- function(input, output, session) {
   
   
   ##Functions for Panel 2 here
+  output$gp_facts_table <- renderTable({
+    facts <- subset(calendar, calendar$"GP Name" == input$gp)
+    row.names(facts) <- facts$"GP Name"
+    facts <-
+      facts |> select(
+        "Round",
+        "Race Date",
+        "Country",
+        "City",
+        "Circuit Name",
+        "Circuit Length(km)",
+        "Number of Laps",
+        "Race Distance(km)",
+        "Turns",
+        "DRS Zones"
+      )
+    transpose <-
+      data.frame(t(facts)) |>
+      rownames_to_column("Circuit")
+    transpose
+  })
+  
+  output$track_layout <- renderImage({
+    filename <- normalizePath(file.path('./www/tracks',
+                                        paste(input$gp, '.png', sep='')))
+    
+    # Return a list containing the filename and alt text
+    list(src = filename,
+         alt = paste(input$gp))
+    
+  }, deleteFile = FALSE)
+  imageOutput("track_layout", height = "100px")
```

Comment:
I don't think we need to call `imageOutput` here since that is called in the ui function.

Reply:
Oh my bad I deleted that but apparently didn't save! Will solve